cve_bin_tool/cli.py

```diff
@@ -53,6 +53,7 @@
 from cve_bin_tool.output_engine import OutputEngine
 from cve_bin_tool.package_list_parser import PackageListParser
 from cve_bin_tool.sbom_manager import SBOMManager
+from cve_bin_tool.sources import curl, nvd
```

Comment:
I think we'd discussed changing this to be `data_sources` rather than `sources` throughout.